[sassc] Build error on x64-mingw-dynamic

Package: sassc:x64-mingw-dynamic@3.6.2

**Host Environment**

- Host: x64-windows
- Compiler: GNU 13.1.0
-    vcpkg-tool version: 2024-08-01-fd884a0d390d12783076341bd43d77c3a6a15658
    vcpkg-scripts version: 0f88ecb85 2024-08-31 (22 hours ago)

**To Reproduce**

`vcpkg install gtkmm`

**Failure logs**

```
-- Using cached sass-sassc-3.6.2.tar.gz.
-- Cleaning sources at C:/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Users/btd/.vcpkg-clion/vcpkg/downloads/sass-sassc-3.6.2.tar.gz
-- Applying patch remove_compiler_flags.patch
-- Using source at C:/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean
-- Found external ninja('1.12.0').
-- Getting CMake variables for x64-mingw-dynamic
-- Using cached msys2-autoconf-wrapper-20221207-1-any.pkg.tar.zst.
-- Using cached msys2-automake-wrapper-20221207-2-any.pkg.tar.zst.
-- Using cached msys2-binutils-2.42-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libtool-2.4.7-3-x86_64.pkg.tar.zst.
-- Using cached msys2-make-4.4.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-pkgconf-2.1.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst.
-- Using cached msys2-bash-5.2.026-1-x86_64.pkg.tar.zst.
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst.
-- Using cached msys2-file-5.45-1-x86_64.pkg.tar.zst.
-- Using cached msys2-gawk-5.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst.
-- Using cached msys2-gzip-1.13-1-x86_64.pkg.tar.zst.
-- Using cached msys2-diffutils-3.10-1-x86_64.pkg.tar.zst.
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.5.3-3-x86_64.pkg.tar.zst.
-- Using cached msys2-autoconf2.71-2.71-3-any.pkg.tar.zst.
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst.
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libintl-0.22.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-zlib-1.3.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-tar-1.35-2-x86_64.pkg.tar.zst.
-- Using cached msys2-findutils-4.10.0-2-x86_64.pkg.tar.zst.
-- Using cached msys2-gmp-6.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-gcc-libs-13.3.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst.
-- Using cached msys2-liblzma-5.6.2-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libzstd-1.5.6-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libreadline-8.2.010-1-x86_64.pkg.tar.zst.
-- Using cached msys2-mpfr-4.2.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libpcre-8.45-4-x86_64.pkg.tar.zst.
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys2-perl-5.38.2-2-x86_64.pkg.tar.zst.
-- Using cached msys2-ncurses-6.5-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libxcrypt-4.4.36-1-x86_64.pkg.tar.zst.
-- Using msys root at C:/Users/btd/.vcpkg-clion/vcpkg/downloads/tools/msys2/212eb2bc95cd8a82
-- Generating configure for x64-mingw-dynamic
-- Finished generating configure for x64-mingw-dynamic
-- Using cached msys2-mingw-w64-x86_64-pkgconf-1~2.2.0-1-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.5.3-3-x86_64.pkg.tar.zst.
-- Using msys root at C:/Users/btd/.vcpkg-clion/vcpkg/downloads/tools/msys2/1e74ca60daa10104
-- Configuring x64-mingw-dynamic-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: C:/Users/btd/.vcpkg-clion/vcpkg/downloads/tools/msys2/212eb2bc95cd8a82/usr/bin/bash.exe --noprofile --norc --debug -c "V=1 CPP='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -E' CC='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe' CC_FOR_BUILD='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe' CPP_FOR_BUILD='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -E' CXX_FOR_BUILD='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/g++.exe' CXX='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/g++.exe' RC='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/windres.exe' WINDRES='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/windres.exe' AR='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/ar.exe' LD='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/ld.exe -verbose' RANLIB='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/ranlib.exe' OBJDUMP='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/objdump.exe' STRIP='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/strip.exe' NM='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/nm.exe' DLLTOOL='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/dlltool.exe' CCAS=':' AS=':' ./../src/3.6.2-bebe44ff8c.clean/configure --build=x86_64-pc-mingw32 \"--with-libsass-include='C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/include'\" \"gl_cv_double_slash_root=yes\" \"ac_cv_func_memmove=yes\" \"--disable-silent-rules\" \"--verbose\" \"--enable-shared\" \"--disable-static\" \"--with-libsass-lib='C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib/libsass.dll.a'\" \"--prefix=/C/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug\" \"--bindir=\\${prefix}/../tools/sassc/debug/bin\" \"--sbindir=\\${prefix}/../tools/sassc/debug/sbin\" \"--libdir=\\${prefix}/lib\" \"--includedir=\\${prefix}/../include\" \"--datarootdir=\\${prefix}/share/sassc\""
    Working Directory: C:/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/x64-mingw-dynamic-dbg
    Error code: 77
    See logs for more information:
      C:\Users\btd\.vcpkg-clion\vcpkg\buildtrees\sassc\config-x64-mingw-dynamic-dbg-config.log
      C:\Users\btd\.vcpkg-clion\vcpkg\buildtrees\sassc\config-x64-mingw-dynamic-dbg-out.log
      C:\Users\btd\.vcpkg-clion\vcpkg\buildtrees\sassc\config-x64-mingw-dynamic-dbg-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:866 (vcpkg_execute_required_process)
  ports/sassc/portfile.cmake:15 (vcpkg_configure_make)
  scripts/ports.cmake:192 (include)



```

<details><summary>C:\Users\btd\.vcpkg-clion\vcpkg\buildtrees\sassc\config-x64-mingw-dynamic-dbg-err.log</summary>

```
configure: WARNING: use of --with-libsass-lib is discouraged!
  use new syntax: ./configure --with-libsass="prefix"
configure: WARNING: use of --with-libsass-include is discouraged,
  use new syntax: ./configure --with-libsass="prefix"
configure: error: in `/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/x64-mingw-dynamic-dbg':
configure: error: C compiler cannot create executables
See `config.log' for more details
```
</details>

<details><summary>C:\Users\btd\.vcpkg-clion\vcpkg\buildtrees\sassc\config-x64-mingw-dynamic-dbg-out.log</summary>

```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for gcc... C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe
checking whether the C compiler works... no
```
</details>

<details><summary>C:\Users\btd\.vcpkg-clion\vcpkg\buildtrees\sassc\config-x64-mingw-dynamic-dbg-config.log</summary>

```
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by sassc configure na, which was
generated by GNU Autoconf 2.71.  Invocation command line was

  $ ./../src/3.6.2-bebe44ff8c.clean/configure --build=x86_64-pc-mingw32 --with-libsass-include=\''C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/include'\''' gl_cv_double_slash_root=yes ac_cv_func_memmove=yes --disable-silent-rules --verbose --enable-shared --disable-static --with-libsass-lib=\''C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib/libsass.dll.a'\''' --prefix=/C/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug '--bindir=${prefix}/../tools/sassc/debug/bin' '--sbindir=${prefix}/../tools/sassc/debug/sbin' '--libdir=${prefix}/lib' '--includedir=${prefix}/../include' '--datarootdir=${prefix}/share/sassc'

## --------- ##
## Platform. ##
## --------- ##

hostname = DESKTOP-2PD6K1G
uname -m = x86_64
uname -r = 3.5.3.x86_64
uname -s = MSYS_NT-10.0-19045
uname -v = 2024-05-20 12:06 UTC

/usr/bin/uname -p = unknown
/bin/uname -X     = unknown

/bin/arch              = x86_64
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = unknown
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /c/Users/btd/.vcpkg-clion/vcpkg/downloads/tools/powershell-core-7.2.23-windows/
PATH: /usr/bin/
PATH: /c/WINDOWS/system32/
PATH: /c/WINDOWS/
PATH: /c/WINDOWS/system32/Wbem/
PATH: /c/WINDOWS/system32/WindowsPowerShell/v1.0/
PATH: /c/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/
PATH: /c/Program Files/Java/jdk-17.0.3.1/bin/
PATH: /c/Program Files/Common Files/Oracle/Java/javapath/
PATH: /c/Program Files (x86)/Common Files/Oracle/Java/javapath/
PATH: /c/Program Files/Eclipse Adoptium/jdk-8.0.322.6-hotspot/bin/
PATH: /c/WINDOWS/system32/
PATH: /c/WINDOWS/
PATH: /c/WINDOWS/System32/Wbem/
PATH: /c/WINDOWS/System32/WindowsPowerShell/v1.0/
PATH: /c/WINDOWS/System32/OpenSSH/
PATH: /c/Program Files/nodejs/
PATH: /c/Program Files/dotnet/
PATH: /c/Program Files (x86)/NVIDIA Corporation/PhysX/Common/
PATH: /c/Program Files (x86)/Windows Kits/10/Windows Performance Toolkit/
PATH: /c/Program Files/Git/cmd/
PATH: /c/Users/btd/AppData/Local/Programs/Python/Python310/Scripts/
PATH: /c/Users/btd/AppData/Local/Programs/Python/Python310/
PATH: /c/Users/btd/AppData/Local/Microsoft/WindowsApps/
PATH: /c/Users/btd/AppData/Roaming/npm/
PATH: /c/Users/btd/AppData/Local/Microsoft VS Code/bin/
PATH: /c/Users/btd/AppData/Local/JetBrains/Toolbox/scripts/
PATH: /c/Users/btd/AppData/Local/Programs/CLion/bin/ninja/win/x64/
PATH: /c/Users/btd/AppData/Local/Programs/CLion/bin/ninja/win/x64/
PATH: /c/Users/btd/.vcpkg-clion/vcpkg/downloads/tools/msys2/1e74ca60daa10104/mingw64/bin/


## ----------- ##
## Core tests. ##
## ----------- ##

configure:2758: looking for aux files: config.guess config.sub ltmain.sh ar-lib compile missing install-sh
configure:2771:  trying ./../src/3.6.2-bebe44ff8c.clean/script/
configure:2800:   ./../src/3.6.2-bebe44ff8c.clean/script/config.guess found
configure:2800:   ./../src/3.6.2-bebe44ff8c.clean/script/config.sub found
configure:2800:   ./../src/3.6.2-bebe44ff8c.clean/script/ltmain.sh found
configure:2800:   ./../src/3.6.2-bebe44ff8c.clean/script/ar-lib found
configure:2800:   ./../src/3.6.2-bebe44ff8c.clean/script/compile found
configure:2800:   ./../src/3.6.2-bebe44ff8c.clean/script/missing found
configure:2782:   ./../src/3.6.2-bebe44ff8c.clean/script/install-sh found
configure:2938: checking for a BSD-compatible install
configure:3011: result: /usr/bin/install -c
configure:3022: checking whether build environment is sane
configure:3077: result: yes
configure:3236: checking for a race-free mkdir -p
configure:3280: result: /usr/bin/mkdir -p
configure:3287: checking for gawk
configure:3308: found /usr/bin/gawk
configure:3319: result: gawk
configure:3330: checking whether make sets $(MAKE)
configure:3353: result: yes
configure:3383: checking whether make supports nested variables
configure:3401: result: yes
configure:3549: checking whether make supports nested variables
configure:3567: result: yes
configure:3595: WARNING: use of --with-libsass-lib is discouraged!
  use new syntax: ./configure --with-libsass="prefix"
configure:3609: WARNING: use of --with-libsass-include is discouraged,
  use new syntax: ./configure --with-libsass="prefix"
configure:3680: checking for gcc
configure:3712: result: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe
configure:4065: checking for C compiler version
configure:4074: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe --version >&5
gcc.exe (GCC) 13.1.0
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

configure:4085: $? = 0
configure:4074: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -v >&5
Using built-in specs.
COLLECT_GCC=C:\Users\btd\AppData\Local\Programs\CLion\bin\mingw\bin\gcc.exe
COLLECT_LTO_WRAPPER=C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/../libexec/gcc/x86_64-w64-mingw32/13.1.0/lto-wrapper.exe
Target: x86_64-w64-mingw32
Configured with: ../gcc-13.1.0/configure --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 --build=x86_64-alpine-linux-musl --prefix=/win --enable-checking=release --enable-fully-dynamic-string --enable-languages=c,c++ --with-arch=nocona --with-tune=generic --enable-libatomic --enable-libgomp --enable-libstdcxx-filesystem-ts --enable-libstdcxx-time --enable-seh-exceptions --enable-shared --enable-static --enable-threads=posix --enable-version-specific-runtime-libs --disable-bootstrap --disable-graphite --disable-libada --disable-libstdcxx-pch --disable-libstdcxx-debug --disable-libquadmath --disable-lto --disable-nls --disable-multilib --disable-rpath --disable-symvers --disable-werror --disable-win32-registry --with-gnu-as --with-gnu-ld --with-system-libiconv --with-system-libz --with-gmp=/win/makedepends --with-mpfr=/win/makedepends --with-mpc=/win/makedepends
Thread model: posix
Supported LTO compression algorithms: zlib
gcc version 13.1.0 (GCC) 
configure:4085: $? = 0
configure:4074: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -V >&5
gcc.exe: error: unrecognized command-line option '-V'
gcc.exe: fatal error: no input files
compilation terminated.
configure:4085: $? = 1
configure:4074: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -qversion >&5
gcc.exe: error: unrecognized command-line option '-qversion'; did you mean '--version'?
gcc.exe: fatal error: no input files
compilation terminated.
configure:4085: $? = 1
configure:4074: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -version >&5
gcc.exe: error: unrecognized command-line option '-version'
gcc.exe: fatal error: no input files
compilation terminated.
configure:4085: $? = 1
configure:4105: checking whether the C compiler works
configure:4127: C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe -g  -I'C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/include' -LC:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib -L'C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib/libsass.dll.a' conftest.c  -lgetopt -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -Wl,-Bstatic,-luuid,-Bdynamic -lcomdlg32 -ladvapi32 >&5
/usr/bin/ld: cannot find -lgetopt: No such file or directory
collect2.exe: error: ld returned 1 exit status
configure:4131: $? = 1
configure:4171: result: no
configure: failed program was:
| /* confdefs.h */
| #define PACKAGE_NAME "sassc"
| #define PACKAGE_TARNAME "sassc"
| #define PACKAGE_VERSION "na"
| #define PACKAGE_STRING "sassc na"
| #define PACKAGE_BUGREPORT "support@moovweb.com"
| #define PACKAGE_URL ""
| #define PACKAGE "sassc"
| #define VERSION "na"
| /* end confdefs.h.  */
| 
| int
| main (void)
| {
| 
|   ;
|   return 0;
| }
configure:4176: error: in `/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/x64-mingw-dynamic-dbg':
configure:4178: error: C compiler cannot create executables
See `config.log' for more details

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_env_CCC_set=
ac_cv_env_CCC_value=
ac_cv_env_CC_set=set
ac_cv_env_CC_value=C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe
ac_cv_env_CFLAGS_set=set
ac_cv_env_CFLAGS_value=-g
ac_cv_env_CPPFLAGS_set=
ac_cv_env_CPPFLAGS_value=
ac_cv_env_CXXCPP_set=
ac_cv_env_CXXCPP_value=
ac_cv_env_CXXFLAGS_set=set
ac_cv_env_CXXFLAGS_value=-g
ac_cv_env_CXX_set=set
ac_cv_env_CXX_value=C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/g++.exe
ac_cv_env_LDFLAGS_set=set
ac_cv_env_LDFLAGS_value=-LC:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib
ac_cv_env_LIBS_set=set
ac_cv_env_LIBS_value=' -lgetopt -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -Wl,-Bstatic,-luuid,-Bdynamic -lcomdlg32 -ladvapi32'
ac_cv_env_LT_SYS_LIBRARY_PATH_set=
ac_cv_env_LT_SYS_LIBRARY_PATH_value=
ac_cv_env_build_alias_set=set
ac_cv_env_build_alias_value=x86_64-pc-mingw32
ac_cv_env_host_alias_set=
ac_cv_env_host_alias_value=
ac_cv_env_target_alias_set=
ac_cv_env_target_alias_value=
ac_cv_func_memmove=yes
ac_cv_path_install='/usr/bin/install -c'
ac_cv_path_mkdir=/usr/bin/mkdir
ac_cv_prog_AWK=gawk
ac_cv_prog_ac_ct_CC=C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe
ac_cv_prog_make_make_set=yes
am_cv_make_support_nested_variables=yes
gl_cv_double_slash_root=yes

## ----------------- ##
## Output variables. ##
## ----------------- ##

ACLOCAL='${SHELL} '\''/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean/script/missing'\'' aclocal-1.16'
AMDEPBACKSLASH=''
AMDEP_FALSE=''
AMDEP_TRUE=''
AMTAR='$${TAR-tar}'
AM_BACKSLASH='\'
AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
AM_DEFAULT_VERBOSITY='1'
AM_V='$(V)'
AR='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/ar.exe'
AUTOCONF='${SHELL} '\''/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean/script/missing'\'' autoconf'
AUTOHEADER='${SHELL} '\''/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean/script/missing'\'' autoheader'
AUTOMAKE='${SHELL} '\''/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean/script/missing'\'' automake-1.16'
AWK='gawk'
CC='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe'
CCDEPMODE=''
CFLAGS='-g'
COMPILER_IS_MINGW32_FALSE=''
COMPILER_IS_MINGW32_TRUE=''
CPPFLAGS=' -I'\''C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/include'\'''
CSCOPE='cscope'
CTAGS='ctags'
CXX='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/g++.exe'
CXXCPP=''
CXXDEPMODE=''
CXXFLAGS='-g'
CYGPATH_W='cygpath -w'
DEFS=''
DEPDIR=''
DLLTOOL='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/dlltool.exe'
DLLWRAP=''
DSYMUTIL=''
DUMPBIN=''
ECHO_C=''
ECHO_N='-n'
ECHO_T=''
EGREP=''
ENABLE_COVERAGE_FALSE=''
ENABLE_COVERAGE_TRUE=''
ETAGS='etags'
EXEEXT=''
FGREP=''
FILECMD=''
GCOV=''
GREP=''
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
LD='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/ld.exe -verbose'
LDFLAGS='-LC:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib -L'\''C:/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug/lib/libsass.dll.a'\'''
LIBOBJS=''
LIBS=' -lgetopt -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -Wl,-Bstatic,-luuid,-Bdynamic -lcomdlg32 -ladvapi32'
LIBTOOL=''
LIPO=''
LN_S=''
LTLIBOBJS=''
LT_SYS_LIBRARY_PATH=''
MAKEINFO='${SHELL} '\''/c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean/script/missing'\'' makeinfo'
MANIFEST_TOOL=''
MKDIR_P='/usr/bin/mkdir -p'
NM='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/nm.exe'
NMEDIT=''
OBJDUMP='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/objdump.exe'
OBJEXT=''
OTOOL64=''
OTOOL=''
PACKAGE='sassc'
PACKAGE_BUGREPORT='support@moovweb.com'
PACKAGE_NAME='sassc'
PACKAGE_STRING='sassc na'
PACKAGE_TARNAME='sassc'
PACKAGE_URL=''
PACKAGE_VERSION='na'
PATH_SEPARATOR=':'
RANLIB='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/ranlib.exe'
SED=''
SET_MAKE=''
SHELL='/bin/sh'
STRIP='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/strip.exe'
VERSION='na'
WINDRES='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/windres.exe'
ac_ct_AR=''
ac_ct_CC='C:/Users/btd/AppData/Local/Programs/CLion/bin/mingw/bin/gcc.exe'
ac_ct_CXX=''
ac_ct_DUMPBIN=''
am__EXEEXT_FALSE=''
am__EXEEXT_TRUE=''
am__fastdepCC_FALSE=''
am__fastdepCC_TRUE=''
am__fastdepCXX_FALSE=''
am__fastdepCXX_TRUE=''
am__include=''
am__isrc=' -I$(srcdir)'
am__leading_dot='.'
am__nodep=''
am__quote=''
am__tar='$${TAR-tar} chof - "$$tardir"'
am__untar='$${TAR-tar} xf -'
bindir='${prefix}/../tools/sassc/debug/bin'
build='x86_64-pc-mingw32'
build_alias='x86_64-pc-mingw32'
build_cpu=''
build_os=''
build_vendor=''
datadir='${datarootdir}'
datarootdir='${prefix}/share/sassc'
docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
dvidir='${docdir}'
exec_prefix='NONE'
host=''
host_alias=''
host_cpu=''
host_os=''
host_vendor=''
htmldir='${docdir}'
includedir='${prefix}/../include'
infodir='${datarootdir}/info'
install_sh='${SHELL} /c/Users/btd/.vcpkg-clion/vcpkg/buildtrees/sassc/src/3.6.2-bebe44ff8c.clean/script/install-sh'
libdir='${prefix}/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${datarootdir}/man'
mkdir_p='$(MKDIR_P)'
oldincludedir='/usr/include'
pdfdir='${docdir}'
prefix='/C/Users/btd/.vcpkg-clion/vcpkg/installed/x64-mingw-dynamic/debug'
program_transform_name='s,x,x,'
psdir='${docdir}'
runstatedir='${localstatedir}/run'
sbindir='${prefix}/../tools/sassc/debug/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias=''

## ----------- ##
## confdefs.h. ##
## ----------- ##

/* confdefs.h */
#define PACKAGE_NAME "sassc"
#define PACKAGE_TARNAME "sassc"
#define PACKAGE_VERSION "na"
#define PACKAGE_STRING "sassc na"
#define PACKAGE_BUGREPORT "support@moovweb.com"
#define PACKAGE_URL ""
#define PACKAGE "sassc"
#define VERSION "na"

configure: exit 77
```
</details>

--- a/ports/sassc/portfile.cmake
+++ b/ports/sassc/portfile.cmake
@@ -1,7 +1,9 @@
+set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO sass/sassc
-    REF 3.6.2
+    REF "${VERSION}"
     SHA512 fff3995ce8608bdaed5f4f1352ae4f1f882de58663b932c598d6168df421e4dbf907ec0f8caebb1e56490a71ca11105726f291b475816dd53e705bc53121969f
     HEAD_REF master
     PATCHES remove_compiler_flags.patch
@@ -9,26 +11,23 @@ vcpkg_from_github(
 
 find_library(LIBSASS_DEBUG sass PATHS "${CURRENT_INSTALLED_DIR}/debug/lib/" NO_DEFAULT_PATH)
 find_library(LIBSASS_RELEASE sass PATHS "${CURRENT_INSTALLED_DIR}/lib/" NO_DEFAULT_PATH)
-if(VCPKG_TARGET_IS_WINDOWS)
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
     set(ENV{LIBS} "$ENV{LIBS} -lgetopt")
 endif()
 vcpkg_configure_make(
     SOURCE_PATH "${SOURCE_PATH}"
     AUTOCONFIG
     OPTIONS
-        --with-libsass-include='${CURRENT_INSTALLED_DIR}/include'
+        "--with-libsass-include='${CURRENT_INSTALLED_DIR}/include'"
     OPTIONS_DEBUG
-        --with-libsass-lib='${LIBSASS_DEBUG}'
+        "--with-libsass-lib='${LIBSASS_DEBUG}'"
     OPTIONS_RELEASE
-        --with-libsass-lib='${LIBSASS_RELEASE}'
+        "--with-libsass-lib='${LIBSASS_RELEASE}'"
 )
 vcpkg_install_make(MAKEFILE GNUmakefile)
 vcpkg_fixup_pkgconfig()
 vcpkg_copy_pdbs()
 
 vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/${PORT}/bin")
-# Handle copyright
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
-
-set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)

--- a/ports/sassc/vcpkg.json
+++ b/ports/sassc/vcpkg.json
@@ -1,10 +1,14 @@
 {
   "name": "sassc",
   "version": "3.6.2",
+  "port-version": 1,
   "description": "SassC is a wrapper around libsass (http://github.com/sass/libsass) used to generate a useful command-line application that can be installed and packaged for several operating systems.",
   "homepage": "https://github.com/sass/sassc",
   "dependencies": [
-    "getopt",
+    {
+      "name": "getopt",
+      "platform": "windows & !mingw"
+    },
     "libsass"
   ]
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8010,7 +8010,7 @@
     },
     "sassc": {
       "baseline": "3.6.2",
-      "port-version": 0
+      "port-version": 1
     },
     "saucer": {
       "baseline": "2.3.0",

--- a/versions/s-/sassc.json
+++ b/versions/s-/sassc.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "937f959ed298caa217edd7352d5743b073ff27f7",
+      "version": "3.6.2",
+      "port-version": 1
+    },
+    {
       "git-tree": "ac068d38115fa4710289d76d3a02a1b6c2650785",
       "version": "3.6.2",
       "port-version": 0

